projects/online/online/utils/pe.py

```diff
@@ -91,54 +121,29 @@ def postprocess_samples(
         idx = inference_params.index(param)
         posterior[param] = samples.T[idx].flatten()
 
+    # TODO remove
+    posterior["phi"] = ra
```

Comment:
Remind me why we need both `posterior["phi"] = ra` and `posterior["ra"] = ra`?

Reply:
This is one of those things I was talking about that should get fixed on the amplfi side. The `Result.to_skymap` function uses `posterior["phi"]` for making the skymap

Reply:
Got it, okay